src/utils/operations/parseAmount.ts

```diff
@@ -0,0 +1,7 @@
+import { DECIMALS } from 'constants/index';
+import { nominate } from './nominate';
+
+export function parseAmount(amount: string, numDecimals: number = DECIMALS) {
+  console.warn();
```

Comment:
see here that you are using nominate as primary function and will print warnings. Move nominate logic here

Reply:
you are right.